_sc2data/Mods/ArchipelagoPlayer.SC2Mod/Base.SC2Data/GameData/ValidatorData.xml

```diff
@@ -376,17 +801,7 @@
         <CombineArray value="AP_IsVanillaEngineeringBay"/>
         <CombineArray value="AP_IsAPEngineeringBay"/>
     </CValidatorCombine>
-    <CValidatorUnitType id="AP_IsAPMedivac">
-        <Value value="AP_Medivac"/>
-    </CValidatorUnitType>
-    <CValidatorUnitType id="AP_IsVanillaMedivac">
-        <Value value="Medivac"/>
-    </CValidatorUnitType>
-    <CValidatorCombine id="IsMedivac" parent="CValidatorCombine">
-        <!-- Override -->
-        <CombineArray value="AP_IsVanillaMedivac"/>
-        <CombineArray value="AP_IsAPMedivac"/>
-    </CValidatorCombine>
+    <!-- Override -->

```

Comment:
This shouldn't be removed

Reply:
That might have been an accident as I was using it as a stencil for other Validators, then removed them. The original was apparently caught int the crossfire